ui/pages/home/home.component.js

```diff
@@ -312,6 +312,7 @@ export default class Home extends PureComponent {
           <ActionableMessage
             type="success"
             className="home__new-network-notification"
+            autoHideTime={5000}
```

Comment:
Nit: could we use the time constants provided by https://github.com/MetaMask/metamask-extension/blob/develop/shared/constants/time.ts here? (`SECOND * 5`)